kuka_kr10_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr10_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR 10 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 10 manipulators. This currently includes the R1100 sixx only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 10 R1100 sixx - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      in the <em>KUKA Roboter GmbH - KR AGILUS sixx - With W and C Variants -
+      Specification</em> version <em>Spez KR AGILUS sixx V12, 26.03.2015</em>.
+      All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise (ie: no support for high speed joints,
+      extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author>Christopher Schindlbeck (University of Hanover)</author>
+  <maintainer email="schindlbeck@imr.uni-hannover.de">Christopher Schindlbeck (University of Hanover)</maintainer>
```

Comment:
Can you please put back my name and email address as a maintainer? Yours can stay as well, but we require all our packages to list at least one core maintainer of ROS-Industrial as well.